File: tools/devctx/fixtures/formats/SampleService.php
<?php

namespace App\Services;

use App\Models\User;
use App\Contracts\UserServiceInterface;

interface UserServiceContract
{
    public function createUser($email);
}

trait Loggable
{
    public function log($message) {}
}

abstract class UserRole
{
    const ADMIN = 'admin';
    const MEMBER = 'member';
}

class SampleService implements UserServiceContract
{
    use Loggable;

    public function createUser($email)
    {
        return new User($email);
    }

    public function deleteUser($id)
    {
        // delete logic
    }
}

function helperFunction()
{
    return "helper";
}

File: tools/devctx/fixtures/formats/SampleService.cs
using System;
using System.Collections.Generic;

namespace Example.Services
{
    public interface IUserService
    {
        Guid CreateUser(string email);
    }

    public enum UserRole
    {
        Admin,
        Member
    }

    public record UserDto(Guid Id, string Email);

    public class SampleService : IUserService
    {
        public Guid CreateUser(string email)
        {
            return Guid.NewGuid();
        }

        public void DeleteUser(Guid id)
        {
            // delete logic
        }
    }
}
